### DISCUSSION TITLE
Errors after v3.5.6 update

### DISCUSSION THREAD

**Discussion by SgtG294** [sn4k3/UVtools#536] [Q&A]
I updated to the latest version and started having odd print fails. There has been no change to settings, ambient room temp, or anything at all. The prints are coming out stringy and flop around like hair. Anyone else experience this? I use Lychee slicer, import to UVTools, then the fixed file goes to my Anycubic M3 Plus.

[Pic 1](https://photos.app.goo.gl/6QMZx6Zssfd2ek5c9) 
[Pic 2](https://photos.app.goo.gl/EUb3SXYxusSZYHYQA)

**Comment by sn4k3** [comment]
Long time ago, nothing changed for Anycubic format within UVtools.

May be related: https://github.com/sn4k3/UVtools/issues/530
